aemedge/styles/styles.css

```diff
@@ -160,7 +160,9 @@ pre {
 }
 
 code {
-  padding: .125em;
```

Comment:
At line 156, remove "code" from the "code, pre" style declaration (so those styles are only for "pre"), and add "font-family: inherit;" here at line 163 under the separate "code" declaration.